firmware/src/peripherals/esp32.rs

```diff
@@ -0,0 +1,60 @@
+use super::Peripherals;
+use crate::aliases::ඞ::DelayConcrete;
+use crate::aliases::ඞ::I2cConcrete;
+
+use fugit::RateExtU32;
+
+use esp32_hal::{
+	clock::{ClockControl, CpuClock},
+	pac,
+	prelude::*,
+	timer::TimerGroup,
+	Rtc,
+};
+
+pub fn get_peripherals() -> Peripherals<I2cConcrete<'static>, DelayConcrete> {
+	let p = pac::Peripherals::take().unwrap();
+
+	let mut system = p.DPORT.split();
+	// The ESP-Wifi module requires 160MHz for cpu clock speeed
+	let clocks =
+		ClockControl::configure(system.clock_control, CpuClock::Clock240MHz).freeze();
+	// Initialize embassy stuff
+	// embassy::init(&clocks);
+
+	// Disable the RTC and TIMG watchdog timers
+	{
+		let mut rtc = Rtc::new(p.RTC_CNTL);
```

Comment:
did you check if this is actually still necessary? Maybe esp hal does this for us?

Reply:
seeing the ``esp-wifi`` example, im guessing it needs to be done manually. But only the RTC, I don't see the need to disable the TIMG watchdog timers in neither esp32 or esp32c3 (should be another PR)